spec/Cabin/Cabin.vspec

```diff
@@ -109,7 +110,7 @@ Door:
   type: branch
   instances:
     - Row[1,2]
-    - ["Left","Right"]
+    - ["Driver","Passenger"]
```

Comment:
I am not convinced it makes sense to call a backdoor "Passenger" door?

Reply:
Think about someone who is delivering use-cases which are driver focused, and now add complexity of LHD and RHD vehicles. "Left" and "Right" is challenging proposal to understand and implement, then you have seat concepts Pos1,2,3.
I understand potential of describing large vehicles, but still our starting point is passenger vehicle.

And I prefer Latin Auriga.

Reply:
Terms like `VDAdata_doorPassengerRear` can be found sometimes

Reply:
Perhaps consider instead referring to the Seat position, being Row<n> and Pos<m>?